modeler.js

```diff
@@ -0,0 +1,70 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+// An action that retrieves a file.
+async function retrieveFile([fileId]) {
+    const modelerAudience = process.env.MODELER_AUDIENCE;
+    const accessToken = await getAccessToken("components", modelerAudience);
+
+    const url = `process.env.MODELER_AUDIENCE/files/${fileId}`;
```

Comment:
After reading the Modeler API docs, it seems like all I need is the a shell script to obtain a token, but with the auth we have set up, am I correct that all we need is the client ID, client secret, and what I've outlined as the audience?

Reply:
Yes, that's correct. I'm not sure what the shell script is doing to obtain a token, but from the perspective of the CLI that we're building, passing the id/secret/audience to the auth endpoint is the typical approach.

Reply:
There are two issues with this line, I think: 

1. It's going to use the text `process.env.MODELER_AUDIENCE` verbatim, instead of injecting the value of that variable. 
2. I'd expect it to be injecting the `MODELER_BASE_URL` variable, not the `MODELER_AUDIENCE`. 

I think the line should look like this, assuming the environment variables were properly configured: 

```suggestion
    const url = `${process.env.MODELER_BASE_URL}/files/${fileId}`;
```